Cargo.toml

```diff
@@ -8,6 +8,6 @@ description = "A library for programatically filling out pdf forms"
 license = "MIT"
 
```

Comment:
I suppose we should be at version 0.3.0, right?

Also, you should probably add yourself and this malte-v to the author list.

Reply:
Yes, I'll do it now :-)